Source/MQTTnet/Internal/AsyncQueue.cs

```diff
@@ -22,7 +22,11 @@ public sealed class AsyncQueue<TItem> : IDisposable
 
         public void Clear()
         {
+#if NETCOREAPP3_1_OR_GREATER
+            _queue.Clear();
```

Comment:
It's cheaper and avoids another allocation.

Note: internally CQ.Clear just drops the segments and creates new ones atomically. So there's an allocation anyway, but we save the one for the CQ itself.